lib/puppet/provider/selinux_login/semanage.rb

```diff
@@ -87,14 +90,14 @@ def self.prefetch(resources)
     instances.each do |provider|
       resource = resources[provider.name]
       if resource
-        unless resource[:selinux_user].to_s == provider.selinux_user && resource[:selinux_login_name].to_s == provider.selinux_login_name || resource.purging?
-          raise Puppet::ResourceError, "Selinux_port['#{resource[:name]}']: title does not match its port and protocol, and a conflicting resource exists"
-        end
+        raise Puppet::ResourceError, "Selinux_port['#{resource[:name]}']: title does not match its port and protocol, and a conflicting resource exists" unless (resource[:selinux_user].to_s == provider.selinux_user && resource[:selinux_login_name].to_s == provider.selinux_login_name) || resource.purging?
```

Comment:
This really is worse. The problem is that we don't have a cop for the max line length (we've disabled that), which means we end up with these horribly long lines.

Reply:
I agreee. I reverted it